Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -1887,75 +1728,7 @@ getFareUpdatedStr diffInDist waitingChargeApplied = do
     false, true -> getVarString (if shorter then FARE_UPDATED_WITH_SHORTER_DIST else FARE_UPDATED_WITH_LONGER_DIST) [ distInKm ]
     true, true -> getVarString (if shorter then FARE_UPDATED_WITH_CHARGES_SHORTER_DIST else FARE_UPDATED_WITH_CHARGES_LONGER_DIST) [ distInKm ]
     false, false -> getString FARE_HAS_BEEN_UPDATED
-
-customerFeedbackPillData :: ST.HomeScreenState -> Array (Array (Array RatingCard.FeedbackItem))
-customerFeedbackPillData state = [ feedbackPillDataWithRating1 state, feedbackPillDataWithRating2 state, feedbackPillDataWithRating3 state, feedbackPillDataWithRating4 state, feedbackPillDataWithRating5 state ]
-
-feedbackPillDataWithRating1 :: ST.HomeScreenState -> Array (Array RatingCard.FeedbackItem)
-feedbackPillDataWithRating1 state =
-  [ [ { id: "6", text: getString RUDE_DRIVER }
-    , { id: "1", text: getString FELT_UNSAFE }
-    ]
-  , [ { id: "6", text: getString RECKLESS_DRIVING }
-    , { id: "6", text: getString DRIVER_CHARGED_MORE }
-    ]
-  , ( [ { id: "15", text: getString TRIP_DELAYED } ] <> acNotWorkingPill state
-    )
-  ]
-
-acNotWorkingPill :: ST.HomeScreenState -> Array RatingCard.FeedbackItem
-acNotWorkingPill state =
-  ( case state.data.ratingViewState.rideBookingRes ^. _serviceTierName of
-      Just serviceTierName ->
-        if ServiceTierCard.showACDetails serviceTierName Nothing then
-          [ { id: "14", text: getString AC_TURNED_OFF } ]
-        else
-          []
-      Nothing -> []
-  )
-
-feedbackPillDataWithRating2 :: ST.HomeScreenState -> Array (Array RatingCard.FeedbackItem)
-feedbackPillDataWithRating2 state = feedbackPillDataWithRating1 state
-
-feedbackPillDataWithRating3 :: ST.HomeScreenState -> Array (Array RatingCard.FeedbackItem)
-feedbackPillDataWithRating3 state =
-  [ [ { id: "8", text: getString UNPROFESSIONAL_DRIVER }
-    , { id: "8", text: getString RASH_DRIVING }
-    ]
-  , [ { id: "8", text: getString DRIVER_CHARGED_MORE }
-    , {id : "11", text : if state.data.vehicleVariant == "AUTO_RICKSHAW" then getString UNCOMFORTABLE_AUTO else if state.data.vehicleVariant == "BIKE" then getString UNCOMFORTABLE_BIKE else getString UNCOMFORTABLE_CAB}
```

Comment:
@ROHIT-43 can we not remove these lines while copying to a different file??
https://github.com/nammayatri/nammayatri/blob/35a559144c6a2bbced1016d40a22ac96db81e2b1/Frontend/ui-customer/src/Helpers/Ride.purs#L123